sycl/source/device_selector.cpp

```diff
@@ -30,28 +30,40 @@ static bool isDeviceOfPreferredSyclBe(const device &Device) {
 
 device device_selector::select_device() const {
   vector_class<device> devices = device::get_devices();
-  int score = -1;
+  // SYCL specification uses -1 to reject a device from selection
+  const int REJECT_DEVICE_SCORE = -1;
```

Comment:
I reckon this should be a class-wide static const as it's also used in `int default_selector::operator()(const device &dev) const`

Reply:
Good point, i've promoted it to class member in  255af95dfd21699b28962971b27dcb1b1e45cb55